src/libutil/sysutil.cpp

```diff
@@ -212,7 +212,7 @@ Sysutil::get_local_time (const time_t *time, struct tm *converted_time)
 #ifdef _MSC_VER
     localtime_s (converted_time, time);
 #else
-    localtime_r (time, converted_time);
+    *converted_time = *localtime(time);
```

Comment:
Will enclose with a specific case for MINGW related. They were part of the changes needed to make it compile. Will strip it off the commits